multiaccounts/migrations/sql/1648646095_image_clock.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE identity_images DROP COLUMN clock;
```

Comment:
hi @siphiuel, Thank you very much for great PR.
We probably not going to use down migrations, but if we ever use this will not work
https://stackoverflow.com/questions/8442147/how-to-delete-or-add-column-in-sqlite